ch_pipeline/hfb/flagging.py

```diff
@@ -0,0 +1,145 @@
+"""HFB Tasks for flagging data
+"""
+import numpy as np
+
+from caput import config
+
+from draco.core import task
+
+from containers import HFBRFIMask
+
+
+class HFBRNTMask(task.SingleTask):
```

Comment:
I think the name of the task should be more descriptive.

Reply:
How about `HFBRadiometerNoiseTestRFIMask`?

Reply:
Converged on `HFBRadiometerRFIFlagging`